tdrs-backend/tdpservice/parsers/schema_defs/ssp/m1.py

```diff
@@ -8,101 +8,402 @@
-                    required=True, validators=[]),
-              Field(item="22B", name='TRANSITION_NBR_MONTHS', type='number', startIndex=81, endIndex=84,
-                    required=True, validators=[]),
-              Field(item="23A", name='OTHER_AMOUNT', type='number', startIndex=84, endIndex=88,
-                    required=True, validators=[]),
-              Field(item="23B", name='OTHER_NBR_MONTHS', type='number', startIndex=88, endIndex=91,
-                    required=True, validators=[]),
-              Field(item="24AI", name='SANC_REDUCTION_AMT', type='number', startIndex=91, endIndex=95,
-                    required=True, validators=[]),
-              Field(item="24AII", name='WORK_REQ_SANCTION', type='number', startIndex=95, endIndex=96,
-                    required=True, validators=[]),
-              Field(item="24AIII", name='FAMILY_SANC_ADULT', type='number', startIndex=96, endIndex=97,
-                    required=True, validators=[]),
-              Field(item="24AIV", name='SANC_TEEN_PARENT', type='number', startIndex=97, endIndex=98,
-                    required=True, validators=[]),
-              Field(item="24AV", name='NON_COOPERATION_CSE', type='number', startIndex=98, endIndex=99,
-                    required=True, validators=[]),
-              Field(item="24AVI", name='FAILURE_TO_COMPLY', type='number', startIndex=99, endIndex=100,
-                    required=True, validators=[]),
-              Field(item="24AVII", name='OTHER_SANCTION', type='number', startIndex=100, endIndex=101,
-                    required=True, validators=[]),
-              Field(item="24B", name='RECOUPMENT_PRIOR_OVRPMT', type='number', startIndex=101, endIndex=105,
-                    required=True, validators=[]),
-              Field(item="24CI", name='OTHER_TOTAL_REDUCTIONS', type='number', startIndex=105, endIndex=109,
-                    required=True, validators=[]),
-              Field(item="24CII", name='FAMILY_CAP', type='number', startIndex=109, endIndex=110,
-                    required=True, validators=[]),
-              Field(item="24CIII", name='REDUCTIONS_ON_RECEIPTS', type='number', startIndex=110, endIndex=111,
-                    required=True, validators=[]),
-              Field(item="24CIV", name='OTHER_NON_SANCTION', type='number', startIndex=111, endIndex=112,
-                    required=True, validators=[]),
-              Field(item="25", name='WAIVER_EVAL_CONTROL_GRPS', type='number', startIndex=112, endIndex=113,
-                    required=True, validators=[]),
-              Field(item="-1", name='BLANK', type='string', startIndex=113, endIndex=150,
-                    required=False, validators=[]),
-          ]
+            model=SSP_M1,
+            preparsing_validators=[
+                validators.hasLength(150),
+            ],
+            postparsing_validators=[],
+            fields=[
```

Comment:
What was the driving change for the reformatting of all these SSP files? Bloats the PR a bit.